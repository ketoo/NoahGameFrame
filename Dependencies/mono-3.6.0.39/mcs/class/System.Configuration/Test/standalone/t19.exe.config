<configuration>
	<appSettings lockAllAttributesExcept="unknown">
		<add key="aloha" value="aloha" />
	</appSettings>
</configuration>