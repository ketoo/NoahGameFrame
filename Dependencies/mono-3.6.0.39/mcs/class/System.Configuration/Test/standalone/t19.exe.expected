Exception raised: System.Configuration.ConfigurationErrorsException
